lte/gateway/c/oai/tasks/amf/amf_sap.h

```diff
@@ -0,0 +1,214 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+#include <sstream>
+#include <thread>
+#ifdef __cplusplus
+extern "C" {
+#endif
+#include "bstrlib.h"
+#ifdef __cplusplus
+};
+#endif
+
+using namespace std;
+#define MIN_GUMMEI 1
+#define MAX_GUMMEI 5
```

Comment:
please comment on why these are the min and max values. what is the purpose.

Reply:
resolved